### PR TITLE
merge #55 - code compliance with code sniffer

### PR DESCRIPTION
**Pull request by mbrenguier**
#55

### CODE REVIEW

File: Library/ContextStorageAdapters/ElasticsearchAdapter.php [hunk 1]
```diff
@@ -177,7 +225,7 @@ private function initCommonRequestParams()
      *
      * @throws ContextStorageException
      */
-    private function initClient()
+    private function _initClient()
```
Comment (jrmbrgs) [inline Library/ContextStorageAdapters/ElasticsearchAdapter.php]: do we really want to follow this convention ?

File: Library/Tasks/AbstractTask.php [hunk 2]
```diff
@@ -144,14 +185,30 @@ protected function sendMessage(string $topicName, Dtos\Message $message) {
...
      * @param string $clientServicesPath      Absolute file path to the service classes
+     *
+     * @return void
      */
-    private function registerClientNS(string $clientServicesNamespace, string $clientServicesPath) {
+    private function registerClientNS(string $clientServicesNamespace, string $clientServicesPath) 
+    {
         $this->getDI()->get('logger')->debug(json_encode(func_get_args()));
         $loader = $this->getDI()->getShared('loader');
+<<<<<<< HEAD
```
Comment (jrmbrgs) [inline Library/Tasks/AbstractTask.php]: Wrong conflict resolution

**Review by jrmbrgs — CHANGES_REQUESTED** [review]

File: Library/ContextStorageAdapters/ElasticsearchAdapter.php (hunk 1 above)
Comment (mbrenguier) [reply Library/ContextStorageAdapters/ElasticsearchAdapter.php]: just forgot to remove this one

Comment (ogarbe) [reply Library/ContextStorageAdapters/ElasticsearchAdapter.php]: no !

File: Library/ContextStorageAdapters/ContextStorageAdapterInterface.php [hunk 3]
```diff
@@ -1,20 +1,37 @@
...
  *
- * @package Disturb\ContextStorageAdapters
+ * @category ContextStorageAdapters
+ * @package  Disturb\ContextStorageAdapters
+ * @author   Alexandre DEFRETIN <adefretin@voyageprive.com>
+ * @license  https://github.com/vpg/disturb/blob/poc/LICENSE MIT Licence
+ * @version  0.1.0
+ * @link     http://example.com/my/bar Documentation of Foo.
  */
 interface ContextStorageAdapterInterface
```
Comment (ogarbe) [inline Library/ContextStorageAdapters/ContextStorageAdapterInterface.php]: in my opinion it's overnaming, it should be named Interface, as the namespace already name it ContextStorageAdapters

File: Library/ContextStorageAdapters/ContextStorageAdapterInterface.php [hunk 4]
```diff
@@ -1,20 +1,37 @@
 <?php
 
+/**
+ * ContextStorageAdapterInterface
+ *
+ * @category ContextStorageAdapters
+ * @package  Disturb\ContextStorageAdapters
+ * @author   Alexandre DEFRETIN <adefretin@voyageprive.com>
+ * @license  https://github.com/vpg/disturb/blob/poc/LICENSE MIT Licence
```
Comment (ogarbe) [inline Library/ContextStorageAdapters/ContextStorageAdapterInterface.php]: this should be merged into master and use master link...

**Pull request merged**